frontend/src/Home.tsx

```diff
@@ -1,9 +1,14 @@
 import React from 'react';
+import GridTiles from './components/gridTemplate';
+import Projectbtn from './components/Button';
```

Comment:
Does this work? Button.tsx is only exporting Button, no Projectbtn. This is a prime example of where you'd use props. By having the display text defined in the Button component, you make it not very reuseable